tests/db-tests/src/test/java/org/apache/activemq/artemis/tests/db/paging/PagingTest.java

```diff
@@ -449,6 +450,12 @@ public void testSimpleCursorIteratorLargeMessage() throws Exception {
 
          message.getBodyBuffer().writerIndex(0);
 
+         message.getBodyBuffer().writeBytes(new byte[numberOfBytes]);
+
+         for (int j = 1; j <= numberOfBytes; j++) {
+            message.getBodyBuffer().writeInt(j);
+         }
```

Comment:
It definitely makes it use the unused numberOfBytes variable, but why write _both_ an array of bytes, and all the ints?